Client/Application/AppLaunchUtil.swift

```diff
@@ -195,17 +206,26 @@ class AppLaunchUtil {
     // MARK: - Application Services History Migration
 
     private func runAppServicesHistoryMigration() {
+        let browserProfile = self.profile as? BrowserProfile
+
         let placesHistory = FxNimbus.shared.features.placesHistory.value()
         FxNimbus.shared.features.placesHistory.recordExposure()
+
         guard placesHistory.migration != .disabled else {
             log.info("Migration disabled, won't run migration")
             return
         }
-        let browserProfile = self.profile as? BrowserProfile
```

Comment:
Curious to know, why was the `browserProfile` moved up there? In the case the `guard` returns it would be created without usage?

Reply:
eeeh no good reason! Probably an artifact of a previous iteration of this 😅 Reverted it back!